courses-fe/src/pages/login/LoginComponent.js

```diff
@@ -1,73 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
 import './style.scss';
 
-class LoginPage extends React.Component {
-    constructor(props) {
-        super(props);
+const LoginPage = (props) => {
+    const [userName, setUserName] = useState('');
+    const [password, setPassword] = useState('');
+    const [submitted, setSubmitted] = useState(false);
 
-        // reset login status
-
-        this.state = {
-            username: '',
-            password: '',
-            submitted: false
-        };
-
-        this.handleChange = this.handleChange.bind(this);
-        this.handleSubmit = this.handleSubmit.bind(this);
-    }
-
-    handleChange(e) {
+    const handleChange = (e) => {
         const { name, value } = e.target;
-        this.setState({ [name]: value });
+        setUserName({ [name]: value });
```

Comment:
aici nu e ok ce ai facut pentru ca ai mai multe functii care iti schimba state-ul...ai setUserName, ai setPassword si trebuie folosite pentru fiecare in parte...de exemplu aici trebuia setat simplu ca si string: setUserName(e.target.value)...la fel trebuie facut si pentru setPassword

Reply:
Uita-te pe register ca ai acolo exact cum ar trebui facut

Reply:
am inteles :)